jupyter_client/kernelspecapp.py

```diff
@@ -92,7 +91,7 @@ def start(self):
             raise
 
 class InstallNativeKernelSpec(BaseIPythonApplication):
-    description = """Install the native kernel spec directory for this Python."""
+    description = """[DEPRECATED] Install the IPython kernel spec directory for this Python."""
```

Comment:
I didn't want to put a warning on the application endpoint when there isn't an alternative.